cpp/src/io/json/json_column.cu

```diff
@@ -195,6 +197,45 @@ reduce_to_column_tree(tree_meta_t& tree,
                                                     : col_ids[parent_node_id];
     });
 
+  // Mixed types in List children go to different columns,
+  // so all immediate children of list column should have same max_row_offsets.
+  //   create list's children max_row_offsets array. (initialize to zero)
+  //   atomicMax on  children max_row_offsets array.
+  //   gather the max_row_offsets from children row offset array.
+  {
+    rmm::device_uvector<NodeIndexT> list_parents_children_max_row_offsets(num_columns, stream);
+    thrust::fill(rmm::exec_policy(stream),
```

Comment:
```suggestion
    thrust::uninitialize_fill(rmm::exec_policy(stream),
```

Reply:
Difference between `thrust::fill` and [thrust::uninitialized_fill](https://thrust.github.io/doc/group__filling_ga7afcf37f1bd2799befe7e93aae7b43a1.html) is that `thrust::fill` uses assignment operator, and `thrust::uninitialized_fill` uses copy constructor. 
For primitive types, it does not make any difference.